sdk/communication/Azure.Communication.CallingServer/src/Models/CallingServerError.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.CallingServer
+{
+    [CodeGenModel("CommunicationError")]
+    public partial class CallingServerError
```

Comment:
Does this need to be public?

Reply:
@navali-msft

Reply:
Removed CallingServerError as it is not used anywhere. Keep the one generated by sdk as internal.